Source/Fuse.ImageTools/Android/AndroidImageUtils.uno

```diff
@@ -42,6 +42,14 @@ namespace Fuse.ImageTools
 			t.start();
 		@}
 
+		public static void GetImageFromBuffer(byte[] bytes, Action<string> onSuccess, Action<string> onFail)
+		{
+			sbyte[] sbytes = new sbyte[bytes.Length];
+			for(var i = 0; i<bytes.Length; i++)
+				sbytes[i] = (sbyte) bytes[i];
+			GetImageFromBuffer(sbytes, onSuccess, onFail);
+		}
```

Comment:
That was an existing implementation, I just moved it to an external method to make it more readable. Would you like me to refactor it inside this pull request or create a separate issue to handle the weird things in current Image Tools implementation?

Reply:
I think having a "fixup weird things in ImageTools"-PR before adding new features would make a lot of sense, yes.

Reply:
@kusma Imho if ImageTools is intended to be supported by 3 platforms (Android, iOS, DotNet) it would be better to implement all 3 versions as is. And then with a next iteration cover them with specification unit tests and align between platforms because them you would have all platform specifics and you can not "overestimate" take features that are supported by 2 platforms, but sadly is not supported by 3rd. So I would vote for adding support with "weird features" and then incrementally refactor and improve the solution as whole. What do you think?